pkg/common/webhook.go

```diff
@@ -28,8 +28,7 @@ const (
 
 	CertSecretName = "fluid-webhook-certs"
 
-	PluginProfileConfigMapName = "webhook-plugins"
-	PluginProfileKeyName       = "pluginsProfile"
+	WebhookPluginFilePath = "/etc/fluid/configmap/webhook_plugins/pluginsProfile"
```

Comment:
How about renaming the filename to `plugins_profile.cfg` or `plugins.profile`?

Reply:
How about using `/etc/fluid/plugins.profile` ?

Reply:
using `/etc/fluid/plugins.profile` now.